TeamCode/src/main/java/org/firstinspires/ftc/teamcode/huskyteers/hardware/Claw.java

```diff
@@ -0,0 +1,68 @@
+package org.firstinspires.ftc.teamcode.huskyteers.hardware;
+
+import androidx.annotation.NonNull;
+
+import com.acmerobotics.dashboard.telemetry.TelemetryPacket;
+import com.acmerobotics.roadrunner.Action;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+import com.qualcomm.robotcore.hardware.Servo;
+
+public class Claw {
+
+    public static final int OPEN_POSITION = 12;
+    public static final int CLOSE_POSITION = 0;
+    public static final double eps = 1e-5;
+    public static final Servo.Direction FORWARD = Servo.Direction.FORWARD;
+    public static final Servo.Direction REVERSE = Servo.Direction.REVERSE;
```

Comment:
Maybe we don't need this right now because we don't use different directions yet